docs/release-notes/FSharp.Compiler.Service.md

```diff
@@ -0,0 +1,21 @@
+---
+category: Release Notes
+categoryindex: 600
+index: 4
+title: FSharp.Compiler.Service
+---
+
+# FSharp.Compiler.Service
+
+## Unreleased
```

Comment:
Can we maybe have "43.8.200 (unreleased)". Because if someone uses preview versions, they will see 43.8.200 and will be able to map to the changes.

We will be able to even check that the version is there. And enforce the check.

Reply:
Good point, I think for `FCS` it should really reflect what is on nuget.org.

So, if there are preview releases they should be included in the changelog.

```md
# FSharp.Compiler.Service

## Unreleased
...

## 43.8.100 - 2023-11-14

...

## 43.8.100-preview.23475.2 - 2023-10-10

...

## 43.8.100-preview.23418.2 - 2023-09-12
```

This is relevant, as a typical motivation to try out a preview FCS is when the last stable FCS was unusable for consumers. If the problem was fixed, there is great value in seeing if that fix made it into the preview.